### DISCUSSION TITLE
is there a way to bring material in a invoice?

### DISCUSSION THREAD

**Discussion by hartwigm** [kimai/kimai#3730] [Questions & Answers] [answered]
Wir benötigen häufig zu den Arbeitszeiten noch die Erfassung von Verbrauchsmaterial. 

Gibt es einen Weg dies bei der Zeiterfassung mit zu erfassen.

Vielen Dank

**Comment by kevinpapst** [comment, marked as answer]
https://www.kimai.org/de/store/keleo-expenses-bundle.html

Kann auf der Demo getestet werden.

**Comment by francwalter** [comment]
Ich habe nur gelegentlich Material, das ich mit auf die Rechnung schreibe, das Plugin für 100.- zu kaufen lohnt sich bei weitem nicht für mich.
Daher trage ich für Material eine zusätzliche Zeiterfassung ein mit 0:00 Dauer und Festpreis (im unteren, aufklappbaren Bereich der Zeiterfassungsmaske). Dazu habe ich auch eine Tätigkeit "Materialbeschaffung" erstellt, die ich dann dafür auswähle.
Beim Festpreis trage ich den Preis des Materials ohne MwSt. ein.
Praktisch wäre natürlich, wenn man da auch Anzahl mit eingeben könnte, damit es ausgerechnet würde, aber ich kann bei mehreren gleich teuren Artikeln einfach in die Beschreibung eintragen zB.: "Vier Netzwerkkabel 5 m" und den Gesamtpreis (-MwSt.), geht auch.

**Comment by kevinpapst** [comment]
Ich mache das hier jetzt zu.

Ideen austauschen finde ich ja prinzipiell okay. Aber ich stecke seit über 7 Jahren meine gesamte Zeit in Kimai und muss davon leben.

Wer sich Workarounds ausdenkt um das Produkt umsonst zu nutzen, der kann das gerne tun. Aber [Freerider sind ein ernstes Problem](https://en.wikipedia.org/wiki/Free-rider_problem), vor allem wenn ein Anbieter sich so weit hinauswagt und das Kern-Produkt schon umsonst anbietet. 

Ich finde das nicht witzig und habe auch keine Lust darüber weiter zu diskutieren.
